### PR TITLE
Provide jobqueue config and example notebook

### PR DESCRIPTION
**Pull request by kathoef**
I have put together a working default Dask jobqueue configuration for JUWELS today (which needs some clean up, but I will do that later). However, as long as https://github.com/dask/dask-jobqueue/issues/382 isn't resolved, opening a Dask jobqueue cluster does not work and we probably should not yet merge this.

(To circumvent the login/compute node interfacing problems, the example notebook was run in an interactive session JupyterLab instance, i.e. JupyterLab also located on a compute node. However, I think this is not a recommended workflow as only full nodes can be allocated on JUWELS and the JupyterLab compute node would be left idling most of the time. Doing calculations locally, i.e. opening a Dask local cluster would avoid this and might be a temporary solution.)

### CODE REVIEW

**Comment by willirath** [comment]
I'm in favor of merging this before the dask issue is solved. After all, this is a working (but not ideal) way of doing this on Juwels for now?

**Comment by kathoef** [comment]
Okay, I agree! :)

File: juwels/01_jobqueue_cluster_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,635 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dask jobqueue example for JUWELS at JSC\n",
+    "covers the following aspects, i.e. how to\n",
+    "* add the JUWELS specific Dask jobqueue configuration\n",
+    "* get overview on available JUWELS compute node resources\n",
+    "* specify batch queue and project budget name\n",
+    "* open, scale and close a default jobqueue cluster\n",
+    "* do an example calculation on larger than memory data"
```
Comment (willirath) [inline juwels/01_jobqueue_cluster_example.ipynb]:
```suggestion
    "* do an example calculation on larger than memory data\n",
    "NOTE: Currently, this **only works** if Jupyter is running **on** a **compute node**."
```

**Review by willirath — CHANGES_REQUESTED** [review]
Looks good. Only one suggestion.

**Review by willirath — APPROVED** [review]

**Pull request merged**